src/main/java/codeu/controller/AdminServlet.java

```diff
@@ -0,0 +1,37 @@
+// Admin Servlet
+
+package codeu.controller;
+
+import java.io.IOException;
+import java.time.Instant;
+import java.util.List;
+import java.util.UUID;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.jsoup.Jsoup;
+import org.jsoup.safety.Whitelist;
+
+/**
+ * AdminServlet 
+ *
+ * This servlet class is for the admin page
+ */
+public class AdminServlet extends HttpServlet {
+  
+  // setup the admin page
```

Comment:
Can you do your comments Javadoc style please? https://google.github.io/styleguide/javaguide.html#s7-javadoc